apps/minaintyg/src/components/Layout/Breadcrumbs.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react'
+import { Outlet, Route, RouterProvider, createMemoryRouter, createRoutesFromChildren } from 'react-router-dom'
+import { Breadcrumbs } from './Breadcrumbs'
+
+it('Should render breadcrumbs', () => {
+  render(
```

Comment:
Could the render be moved out to a function so it's easier to add tests later without duplicating code?

Reply:
Added a `beforeEach` that renders the component and better test descriptions. I think we need to make bigger changes to breadcrumb when navigating in too certificates, so let's keep it simple for now.